brands/README.md

```diff
@@ -1,16 +1,18 @@
 # Brands
 
-After creating an account and having access to the dashboard, the next step is to create a test **Brand** so you can create locations and add them to a Program in order to start tracking transactions.
+After creating an account and having access to the dashboard, the next step is to create a test **Brand** (Merchant) so you can create locations and add them to a Program in order to start tracking transactions. 
 
-The **Brand** (Merchant) object is used to aggregate locations and keep the status of the Brand consent. One of the requirements, when creating a Brand and before adding locations, is to obtain Brand consent from an authorized personnel (Brand User) and to provide the Brand with access to view the transactional data that is being shared.
+>Once you create a **Brand**, it cannot be deleted.
 
-You can only create new locations on live environment from Brands that you have invited to create an account have access to their card linked transactions. On test environment, the consent status is automatically approved so you can continue testing.
+The **Brand** object is used to aggregate locations and keep the status of the Brand consent. 
 
 ## Create Brand
 
 To create a new Brand, go to the **Brands** page on the dashboard, click the **+** icon and enter a name. Optionally, you can add a link to the brand logo.
 
-To automatically request Brand consent, enter the name and email of the Brand User and will send an email inviting the Brand User to create a Brand account at [clo.fidel.uk](https://clo.fidel.uk). By creating an account the Brand User provides consent and will have access to transactions made by cardholders at their participating locations.
+To automatically request Brand consent, enter the name and email of the Brand User and Fidel will send an email inviting the Brand User to create a Brand account at [clo.fidel.uk](https://clo.fidel.uk). By creating an account the Brand User provides consent and will have access to transactions made by cardholders at their participating locations.
+
+>Brand Consent is only required in the Live environment. Test Brands are automatically given Brand consent.
```

Comment:
Let's remove the `>`, and put this sentence in a `<div class="info-box">`

We don't have to capitalise `live` and `test`.

Change `Test Brands are automatically given Brand consent` to `In test environment, the consent status is automatically approved.`

Reply:
Let's add a new paragraph:

`If you're funding your own offers, or you've received direct authorisation from a brand, you can now auto-approve the brand's consent – [see here](https://releasenotes.fidel.uk/auto-approve-brand-consent-92004) for more information.`